### ISSUE TITLE
Loss of information in LDAPOperationResult exception with REUSABLE strategy

### ISSUE THREAD

**Issue by nitmir** [cannatag/ldap3#702]
https://github.com/cannatag/ldap3/blob/08810e435392a1f1a96d5e65bb444b9cb9e96ae3/ldap3/strategy/reusable.py#L263-L265

This cause the exception raises in the main thead to only have `result` to set to a string instead of an integer and all other parameters (`description`, `dn`, `message`, `response_type`, `response`) of the exception set to None.

Is there a reason to not simply do 
```
pool._incoming[counter] = (e, None, None) 
```
which seems to work with my python 3.5 ?

Otherwise, is it possible to as least do something like
```
pool._incoming[counter] = (copy.deepcopy(e), None, None)
```
(which also seems to work with my python 3.5).

in order to keep the the exception informations ?

**Comment by cannatag** [comment]
Hi, the casting is to be sure that in the exception there is no reference to any "mutable" object. I'm not sure if this is the case, so I've changed the code as you suggested. 

Can you try the code in dev?
Let me know if this works as intended.

Bye,
Giovanni

**Comment by cannatag** [comment]
released in v2.6.1
